test/regex/bytes/cast.good

```diff
@@ -1,3 +1,9 @@
+cast.chpl:8: warning: Casting strings to regex is deprecated. Use regex.compile(string) instead.
+cast.chpl:9: warning: Casting bytes to regex is deprecated. Use regex.compile(bytes) instead.
```

Comment:
I might remove lines 8-15 from this test since the deprecated cast is still tested in /deprecated/regexTertiary